### PR TITLE
Go 1.11 compatibility

### PR DESCRIPTION
**Pull request by divoxx**
With go 1.11, the project's folder does not need to match the application name or import path anymore. The current version uses the path basename as the application name, which causes `--run` to fail to run the application with a `project not found` error.

This PR changes `run` to use `go list` to get the import path and then extract the name of the application from it.

### CODE REVIEW

File: realize_test.go [hunk 1]
```diff
@@ -3,7 +3,7 @@ package main
 import (
 	"bytes"
 	"errors"
-	"github.com/oxequa/realize/realize"
+	"github.com/divoxx/realize/realize"
```
Comment (divoxx) [inline realize_test.go]: This needs to be updated prior to merge, if to be merged.

File: realize.go [hunk 2]
```diff
@@ -8,8 +8,8 @@ import (
 	"strings"
 	"time"
 
+	"github.com/divoxx/realize/realize"
```
Comment (divoxx) [inline realize.go]: This needs to be updated prior to merge, if to be merged.

**Comment by frederikhors** [comment]
Any news on this?

**Comment by erikdubbelboer** [comment]
Looks like the project is abandoned yes. I have started a fork here https://github.com/erikdubbelboer/realize with all the pull requests that look good such as this one merged.

**Pull request closed without merging**